src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -19,92 +19,89 @@ import aiExplainThisErrorPopUp from 'images/ai_screenshot-crop_explain-this-erro
+ <TabsPageItem id="1`">
+ Our AI chat experience can help get you started using New Relic. If you need help organizing your data and setting up alerts, NRAI can act as a guide throughout your onboarding process. Ask our AI questions like:
 
 * How can I use Terraform to create dashboards? 
 * How can I improve my alerts setup?
 * How can I instrument AWS?
-    </TabsPageItem>
+ </TabsPageItem>
 
-    <TabsPageItem id="2">
-    If you want to get to the root cause of latency issues for your app, here are some good questions to ask:
+ <TabsPageItem id="2">
+ If you want to get to the root cause of latency issues for your app, here are some good questions to ask:
 
 * What are some recent incidents in my system?
 * Which infrastructure hosts are showing signs of degradation over the past hour?
-    </TabsPageItem>
+ </TabsPageItem>
 
-    <TabsPageItem id="3">
-    Our AI can also be your partner in observability maturity. If you're responsible for ensuring that your app can handle spikes for a big e-commerce event like Black Friday, it's important to understand the state of your system so you can improve it _ahead of time_. Ask our AI:
+ <TabsPageItem id="3">
+ Our AI can also be your partner in observability maturity. If you're responsible for ensuring that your app can handle spikes for a big e-commerce event like Black Friday, it's essential to understand the state of your system so you can improve it _ahead of time_. Ask our AI:
 
 * How can I improve the performance of my infrastructure?
 * Were unique user sessions up or down last week compared to the week before?
-    </TabsPageItem>
-    </TabsPages>
+ </TabsPageItem>
+ </TabsPages>
 </Tabs>
 
-### Get help creating NRQL queries [#NRQL-console]
+### Create NRQL queries [#NRQL-console]
 
-[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool you can use to gain insights into systems, but when you're in a hurry you might not have time to create the right query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. You can ask questions like, "How is this service performing?" or "How has this service been performing over the past 24 hours?"
+[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool to gain insights into systems. Still, when you're in a hurry, you might not have time to create the correct query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. You can ask questions like, "How is this service performing?" or "How has this service been performing over the past 24 hours?"
 
 <img
-        title="Screenshot showing the Ask AI button in the New Relic platform"
-        alt="ask AI button"
-        src={askgrok}
-    />
-
+ title="Screenshot showing the Ask AI button in the New Relic platform"
+ alt="ask AI button"
```

Comment:
```suggestion
 title="Ask AI button"
 alt="Screenshot showing the Ask AI button in the New Relic platform"
```

The alt text should decribe the image to an e-reader, whereas the title appears when you hover over an image on the docs site.

Reply:
Thanks for catching this! It was a mistake.